Content.Server/Crayon/CrayonComponent.cs

```diff
@@ -24,5 +24,11 @@ public sealed partial class CrayonComponent : SharedCrayonComponent
         [ViewVariables(VVAccess.ReadWrite)]
         [DataField("deleteEmpty")]
         public bool DeleteEmpty = true;
+
+        [ViewVariables(VVAccess.ReadWrite)]
+        public float Rotation { get; set; }
+
+        [ViewVariables(VVAccess.ReadWrite)]
+        public bool PreviewMode { get; set; }
```

Comment:
```suggestion
        [DataField]
        public float Rotation;

        [DataField]
        public bool PreviewMode;
```
also given that these are also seen on the client component, i would put these in shared + network them